app/renderer/components/frame/guestInstanceRenderer.js

```diff
@@ -0,0 +1,205 @@
+const React = require('react')
+const {StyleSheet, css} = require('aphrodite/no-important')
+
+// Components
+const ReduxComponent = require('../reduxComponent')
+// comment out to not use pooled webview (2x webviews):
```

Comment:
If time allows, it would be much better to check for an environment variable instead of having the notes about commenting in here. Something like:
```
let WebviewDisplay
if (process.env.USE_POOLED_WEBVIEW === false) {
  WebviewDisplay = require('../../webviewDisplay')
} else {
  WebviewDisplay = require('../../pooledWebviewDisplay')
}
```